lib/linker.version

```diff
@@ -90,4 +90,5 @@ MOSQ_1.5 {
 		mosquitto_sub_topic_check2;
 		mosquitto_topic_matches_sub2;
 		mosquitto_connect_with_flags_callback_set;
+		mosquitto_max_pub_queue_messages_set;
```

Comment:
Thanks for spotting this important file. The new function will have to be in a new section, could you create it at the bottom of the file like:
```
MOSQ_1.6 {
    global:
        <function>
} MOSQ_1.5;
```